<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveSampleDialog</class>
 <widget class="QDialog" name="SaveSampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>160</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sound sample extracted.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Would you like to save it?</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="slot1RadioButton">
        <property name="text">
         <string>Slot 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="slot2RadioButton">
        <property name="text">
         <string>Slot 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="slot3RadioButton">
        <property name="text">
         <string>Slot 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="slot4RadioButton">
        <property name="text">
         <string>Slot 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveSampleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveSampleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
